client/src/partials/primary_cash/expense/multi_payroll.html

```diff
@@ -63,6 +92,12 @@
       <div class="panel panel-default">
         <div class="panel-heading" >
           <b>{{ "PRIMARY_CASH.EXPENSE.PAYABLE_EMPLOYEE" | translate}} {{session.pp.dateFrom | date}} / {{session.pp.dateTo | date}}</b>
+
+          <!-- show loader for GET methods -->
+          <div class="pull-right">
+            <span ng-if="session.state==='loading'">{{ "UTIL.LOADING" | translate}} ...</span>
```

Comment:
Did you know we have a directive for this?  It is `<loading-indicator></loading-indicator>`.  It is exactly what you are trying, except it has a spinning icon with css!